packages/web-components/src/components/ic-stepper/ic-stepper.e2e.ts

```diff
@@ -42,7 +42,7 @@ describe("default variant of ic-stepper component", () => {
 
     const steps = await page.evaluate(() => {
       return Array.from(
-        document.querySelectorAll("ic-step:not(:last-child)")
+        document.querySelectorAll("ic-step > div.step-connect")
```

Comment:
The existing way of doing this was failing the e2e test, returning an array of ["auto", "280px", "auto"], changing it to this seemed to be the only way that the step matched 198px

Reply:
The line here to do the assertion:
```javascript
    steps.forEach((step) => {
      expect(step).toMatch(`${connectorWidth + 48}px`);
    });
```
..is not running. 

Line 45 returns an empty array so the `forEach` is not being doing a loop. We are bypassing this assertion. Let's take out this test and then refactor it on a new ticket as it's out of scope for this ticket.

Reply:
If I change the expected width on line 50 to a random number, the test still passes - so think it must be because it's returning an empty array like @jd239 says which is why that line is not being run. 

Also, we're checking if the whole step is 198px on line 50, so don't think we want a query selector for just the step-connect div - unless we change the expect to just check that the step connector is 150px (in which case, I think we may need to use a shadowRoot with the query selector to get to elements within the ic-step?).

Basically I think we need to refactor this test slightly as it's not quite working / testing what we want :)

Reply:
#366 ticket created 👍